### PR TITLE
add idam id and ccd id to logs

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-3703 - Print CCD & IDAM ID(?) in DN & AOS](https://tools.hmcts.net/jira/browse/DIV-3703)

### CODE REVIEW

File: middleware/feesAndPaymentsMiddleware.js [hunk 1]
```diff
@@ -10,15 +10,18 @@ const getFeeFromFeesAndPayments = feeUrl => {
     return feesAndPaymentsService.get(feeUrl)
       .then(response => {
         // set fee returned from Fees and payments service
-        logger.info(' Fee amount set to ', response.amount);
+        logger.info(logger.wrapWithUserInfo(req, 'Fee amount set to ${response.amount}'));
```
Comment (hosainnet) [inline middleware/feesAndPaymentsMiddleware.js]: The 'Fee amount..' string should be sournded with `

File: test/unit/services/logger.test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  describe('#accessLogger', () => {
+    it('returns access logger middleware that is excutable', () => {
+      const middleware = logger.accessLogger();
+      const res = {
+        statusCode: 200,
+        end: sinon.stub()
+      };
+      middleware(reqWithIdam, res, sinon.stub());
+      res.end();
```
Comment (hosainnet) [inline test/unit/services/logger.test.js]: Is this asserting anything

Comment (dtaylor7) [reply test/unit/services/logger.test.js]: no, but its runs the code and ensures it runs ok, i couldn't find away to assert the log had been called as expected, spent a few hours trying too....

Comment (hosainnet) [reply test/unit/services/logger.test.js]: 👍

**Pull request merged**